### PR TITLE
multitimer: add setting for initial screen

### PR DESCRIPTION
**Pull request by bobrippling**
This also updates `multitimer.json` to hold settings as well as stopwatches.

@frigis1 is this change ok by you? No change in behaviour by default

### CODE REVIEW

**Comment by thyttan** [comment]
My fw2v21.25 bangle starts loading the app, but then nothing seems to happen:

image

**Comment by bobrippling** [comment]
Interesting - I found an old bug where the scroller wasn't saved, which I've fixed but I don't see how that would cause it to crash on startup. Do you still see it? And do you have anything in `multitimer.json` / would you mind sharing the [console output](https://www.espruino.com/ide/) when you start the app?

**Comment by thyttan** [comment]
```
Uncaught Error: Can't read property 'initialScreen' of undefined
 at line 728 col 26 in multitimer.app.js
switch (readAndConvJson().initialScreen) {
```

I don't really understand the error yet, `readAndConvJson()` looks like it should be defined. What I don't find is that it should have a `initialScreen` property. But that's not what it complains about?

File: apps/multitimer/app.js [hunk 1]
```diff
@@ -328,9 +328,21 @@ function editTimer(idx, a) {
   setUI();
 }
 
+function readAndConvJson() {
+  let json = require("Storage").readJSON("multitimer.json", true);
```
Comment (nxdefiant) [inline apps/multitimer/app.js:332]: Probably missing a default value e.g. || {}

**Comment by nxdefiant** [comment]
Yes, the error means that readAndConvJson() returns undefined.

File: apps/multitimer/app.js (hunk 1 above)
Comment (bobrippling) [reply apps/multitimer/app.js]: Thank you!

**Comment by bobrippling** [comment]
Doh - yes I always tested with a file present, hadn't considered that if a user's never created a stopwatch they won't have the file. Pushed the fix (27cfb0e)

**Comment by frigis1** [comment]
@bobrippling I'm afraid I haven't had time at all to update my stuff, so any fixes and updates are not only welcome but also greatly appreciated. Thank you!

**Comment by thyttan** [comment]
Tried it now without running into bugs. Do you feel like it's merge ready?

**Comment by bobrippling** [comment]
Thank you, yes I feel so - think it's taken a Saturday brain to fix it!

**Comment by thyttan** [comment]
Ok - thanks then! :)

**Comment by thyttan** [comment]
Oops - hadn't tried the settings page: 

```
 ____                 _
|  __|___ ___ ___ _ _|_|___ ___
|  __|_ -| . |  _| | | |   | . |
|____|___|  _|_| |___|_|_|_|___|
         |_| espruino.com
 2v21.25 (c) 2023 G.Williams
Multi Timer settings error: Error: Error: Can't read property 'initialScreen' of undefined
>g.dump()
```
image

**Comment by bobrippling** [comment]
Doh! Great spot - pushed a fix in #3226 (quite pleased with that rhyme)

**Pull request merged**